### ISSUE TITLE
Support to import resources

### ISSUE THREAD

**Issue by bronislav** [Yelp/terraform-provider-signalform#18]
Is this in the roadmap or there is API limitations that prevent to implement this feature?

**Comment by poros** [comment]
My answer is based on the assumption you are referring to this https://www.terraform.io/docs/import/index.html

We started to create resources via the SignalFx v2 API directly and only via this provider, so we never had the need to implement import of existing resources. So no, I would say there is no plan but no limitation either, if you are aware that v1 resources cannot be managed via the API.

**Comment by bronislav** [comment]
Yes, you are right. I asked about importing resources created manually to the terraform state. 

I didn't know that v1 resources could not be managed using API. This answers my question. Most of the resources that we potentially want to import, created using v1. 

I just aware that their API doesn't support all the things that supported in the UI.
